scripts/tslint-rule-to-eslint.js

```diff
@@ -43,7 +49,7 @@ const parseAndReadFileContents = async filePath => {
         delete tslintData.rules[tslintRuleName];
         if (Object.keys(tslintData.rules).length === 0) {
             console.log(`\t${tslintFilePath} has no remaining rules; deleting rules property.`);
-            delete tslintFilePath.rules;
+            delete tslintData.rules;
```

Comment:
Is it overplayed to make a snarky comment about how TypeScript would have caught this?

Reply:
This file isn't `// ts-check` or included in a project?

Reply:
I excluded it in `jsconfig.json` because getting types to work in JSDoc with the `tslintData: CommentJSONValue` objects was getting to be a lot of comments. But, I suppose I can just `any`...